code/client/client.cpp

```diff
@@ -197,10 +204,14 @@ int main(int argc, char* argv[]) {
 			//pass the event to the camera
 			gs.input(window.event, controlledCamera);
 		}
-		gs.render();
+		
 
 		// BEGIN ECS SYSTEMS UPDATES 
+		//std::cout << "updating systems\n";
 		exampleEcsSystem.Update(mainScene, 0.0f);
+		//std::cout << "exmple system finished\n";
+		gs.Update(mainScene, 0.0f);
```

Comment:
could you change this variable to be more self-descriptive? maybe change gs to graphicsSystem? it could be hard to understand at first glance

Reply:
*dabs*